src/main/scala/rx/lang/scala/subjects/UnicastSubject.scala

```diff
@@ -0,0 +1,52 @@
+/**
+  * Copyright 2013 Netflix, Inc.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *     http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+package rx.lang.scala.subjects
+
+import rx.annotations.Experimental
+import rx.lang.scala.Subject
+
+/**
+  * A `Subject` variant which buffers events until a single `Subscriber` arrives and replays them to it
+  * and potentially switches to direct delivery once the `Subscriber` caught up and requested an unlimited
+  * amount. In this case, the buffered values are no longer retained. If the `Subscriber`
+  * requests a limited amount, queueing is involved and only those values are retained which
+  * weren't requested by the `Subscriber` at that time.
+  */
+@Experimental
```

Comment:
Putting the `@Experimental` annotation here is fine, but in addition, you could also add a red badge in the scaladoc comment, as in [ErrorDelayingObservable](https://github.com/ReactiveX/RxScala/blob/0.x/src/main/scala/rx/lang/scala/observables/ErrorDelayingObservable.scala).